dev/com.ibm.ws.jbatch.security/src/com/ibm/ws/jbatch/security/impl/WSBatchAuthServiceImpl.java

```diff
@@ -253,27 +337,62 @@ private long authorizedJobStopRestartByInstance(long instanceId)
     	return instanceId;
     }
     
+    private void authorizedWriteOperationAccessCheck(String submitter, long instanceId) 
+            throws NoSuchJobExecutionException, JobSecurityException {
+		Set<String> listOfGroupsForJobID = null;
+		Set<String> listOfGroupsForSubject = null;
+		boolean checkGroupSecurity = false;
+
+		if (isInAnyBatchRole()) {
+			if (this.isAdmin(runAsSubject())) {
+			} else if (this.isSubmitter(runAsSubject())) {
+				boolean submitterRole = this.isSubmitter(runAsSubject());
+				boolean isGroupAdmin = this.isGroupAdmin();
+				boolean isGroupMonitor = this.isGroupMonitor();
+				// if you don't also own the job you can't view it
+				if (batchSecurityHelper.getRunAsUser().equals(submitter)) {
+				} else if (isGroupAdmin) {
```

Comment:
Along with my comments in the read auth method that it might be nice to see explicit `return retVal;` stmts it might also be nice to cut a log/trace about which point we're exiting from.    That's more important actually, for this and the read method.